lib/jwk/keystore.js

```diff
@@ -395,6 +395,11 @@ var JWKStore = function(registry, parent) {
     value: function(props, local) {
       props = props || {};
 
+      // workaround for issues/109
+      if (typeof props.kid === "number") {
```

Comment:
Instead of equaling number, I'd rather have a check for a non-null non-string.  I realize that's more broad, but this feels too targeted